pkgs/development/tools/build-managers/buildbot/default.nix

```diff
@@ -7,10 +7,10 @@
 pythonPackages.buildPythonApplication (rec {
   name = "${pname}-${version}";
   pname = "buildbot";
-  version = "0.9.0rc2";
+  version = "0.9.0rc3";
   src = fetchurl {
     url = "mirror://pypi/b/${pname}/${name}.tar.gz";
-    sha256 = "06xzbjzji3by4hldwr850fc989ymsfl2s8nil21klv2g00dgmcmk";
+    sha256 = "18238n9l0pfivb23csy5lnkx91fcm4jgmhxn03f2nfnp00rlppzb";
   };
 
   buildInputs = with pythonPackages; [
```

Comment:
i believe buildbot fixed their packaging so we can remove lines 62-69 here.

(https://github.com/buildbot/buildbot/blob/90ae62b19db764b8099d308abc1f6d313b0b01f3/master/docs/relnotes/index.rst#fixes)